packages/fx-core/src/core/error.ts

```diff
@@ -128,8 +128,8 @@ export class TaskNotSupportError extends SystemError {
   }
 }
 
-export function FetchSampleError(): UserError {
-  return new UserError("FetchSampleError", "Failed to download sample app", CoreSource);
+export function FetchSampleError(sampleId: string): UserError {
+  return new UserError("FetchSampleError", `Empty zip file for ${sampleId}`, CoreSource);
```

Comment:
why the error message is 'empty zip file xxx'?

Reply:
Updated